src/Sulu/Bundle/AdminBundle/Resources/js/components/Grid/types.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import type {Element} from 'react';
+
+export type Size = 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+
+export type BaseItemProps = {
+    size: Size,
+    spaceBefore: Size,
+    spaceAfter: Size,
+    children: Element<*>,
```

Comment:
Isn't this different for both elements? I'd say the section is only allowed to have other `Item` and `Section` elements, whereby the statement here is only true for the item component.